client/questions_answers/components/HelpfulReport.jsx

```diff
@@ -6,17 +6,22 @@ const HelpfulReport = (props) => {
   let helpfulIds = JSON.parse(localStorage.getItem(props.content_type === 'question' ? 'helpfulQuestions' : 'helpfulAnswers'));
 
   if (props.reported) {
-    renderReportedLink = (<span> Reported </span>);
+    renderReportedLink = (<span> <i>Reported</i> | </span>);
   } else {
-    renderReportedLink = (<button onClick={props.handleReportButtonClick}> Report </button>);
+    renderReportedLink = ( <span>
```

Comment:
Out of curiosity why are you wrapping some elements in a <span> tag?

Reply:
I can't remember. Maybe I was planning to add a class or something? I'll check that out. Thanks!